cscs-checks/apps/lammps/lammps_check.py

```diff
@@ -58,10 +58,10 @@ def __init__(self, **kwargs):
         self.tags |= {'maintenance'}
         self.reference = {
             'dom:gpu': {
-                'perf': (3409, -0.15, None)
+                'perf': (3457.0, -0.05, None)
             },
             'daint:gpu': {
-                'perf': (4880, -0.15, None)
+                'perf': (4691.0, -0.50, None)
```

Comment:
This is worse than before and we never had a problem. You should perhaps remove more outliers from your dataset.